### PR TITLE
fix: task responce changed

### PR DESCRIPTION
**Pull request by sbilevich**

### CODE REVIEW

File: web/src/api/typings/tasks.ts [hunk 1]
```diff
@@ -12,11 +12,18 @@ export type Task = {
         name: string;
         id: number;
     };
-    user_id: string;
+    user: {
+        id: string;
+        name: string;
+    };
```
Comment (Iogsotot) [inline web/src/api/typings/tasks.ts:18]: we have a user object with an id field and a separate user_id field, right? what is the difference between these id?

File: web/src/api/typings/tasks.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { ValidationType } from 'api/typings';
 
-export type Task = {
+export type ApiTask = {
```
Comment (Iogsotot) [inline web/src/api/typings/tasks.ts:3]: Can you clarify for me please: does taskApi mean a task that comes from the backend?

File: web/src/api/typings/tasks.ts [hunk 3]
```diff
@@ -12,11 +12,18 @@ export type Task = {
...
-    user_id: string;
+    user: {
+        id: string;
+        name: string;
+    };
     is_validation: boolean;
     deadline: string;
 };
 
+export type Task = Omit<ApiTask, 'user'> & {
```
Comment (Iogsotot) [inline web/src/api/typings/tasks.ts:23]: What is the difference between entities Task and ApiTask?

**Review by Iogsotot — APPROVED** [review]

File: web/src/api/typings/tasks.ts (hunk 2 above)
Comment (sbilevich) [reply web/src/api/typings/tasks.ts:3]: yes

File: web/src/api/typings/tasks.ts (hunk 1 above)
Comment (sbilevich) [reply web/src/api/typings/tasks.ts:18]: before changes on backend there was user_id, now user object. I created new type for task that comes from backend, so we can take user id from it and use it on front-end without changing Task type

File: web/src/api/typings/tasks.ts (hunk 3 above)
Comment (sbilevich) [reply web/src/api/typings/tasks.ts:23]: Task has user_id, ApiTask user:{id, name}

**Pull request merged**